.eleventy.js

```diff
@@ -88,17 +88,19 @@ module.exports = eleventyConfig => {
   eleventyConfig.addPlugin(syntaxHighlight);
 
   // Add collections
-  locales.forEach(locale => eleventyConfig.addCollection(`blog-${locale}`, collections => {
-    let blogCollection = collections
-      .getFilteredByGlob(`./site/${locale}/blog/*/*.md`)
-      .filter(filterOutDrafts)
-      .reverse();
-    // If we're running inside of Percy then just show the first six blog posts.
-    if (process.env.PERCY_BRANCH) {
-      blogCollection = blogCollection.slice(blogCollection.length - 6);
-    }
-    return blogCollection;
-  }));
+  locales.forEach(locale =>
```

Comment:
This is due to my IDE's auto-formatting, but it's probably a good idea to include it just to keep everything consistent.